#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_fix_418703_dont_use_abbreviated_sfnet_address.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Ondrej Certik reported a resolver issue: http://bugs.debian.org/418703.
## DP: The error seems to be caused by using the abbreviated sf.net URLs. But
## DP: it is possible, that this issue only occurs together with the issue
## DP: described in 01_fix_static_datadir_evaluation.dpatch, because the path
## DP: to the catalog also suffers from this issue.

@DPATCH@
diff -urNad docbook2x-0.8.8~/perl/db2x_xsltproc.pl docbook2x-0.8.8/perl/db2x_xsltproc.pl
--- docbook2x-0.8.8~/perl/db2x_xsltproc.pl	2004-08-18 16:21:52.000000000 +0200
+++ docbook2x-0.8.8/perl/db2x_xsltproc.pl	2007-04-12 16:07:20.000000000 +0200
@@ -110,10 +110,10 @@
     
     if($options->{'stylesheet'} eq 'texi') {
         $options->{'stylesheet'} = 
-            "http://docbook2x.sf.net/latest/xslt/texi/docbook.xsl";
+            "http://docbook2x.sourceforge.net/latest/xslt/texi/docbook.xsl";
     } elsif($options->{'stylesheet'} eq 'man') {
         $options->{'stylesheet'} = 
-            "http://docbook2x.sf.net/latest/xslt/man/docbook.xsl";
+            "http://docbook2x.sourceforge.net/latest/xslt/man/docbook.xsl";
     }
 
     if(scalar(@argv) != 1) {
